src/Resources/views/Error/layout.html.twig

```diff
@@ -136,17 +136,17 @@
                         <div class="text-error">
                             {% block main %}
                                 {% set _block = block('matter') %}
-                                {% if _block is not empty %}
+                                {% if _block|trim is not empty %}
                                     <h2>{{ error.matter }}</h2>
                                     {{ _block|raw }}
                                 {% endif %}
                                 {% set _block = block('howToFix') %}
-                                {% if _block is not empty %}
+                                {% if _block|trim is not empty %}
```

Comment:
They are not, but they should be applied anyway. It is best practice to trim the return value in conjunction with `not empty`.